x-pack/plugins/apm/public/components/shared/aggregated_transactions_callout/index.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiCallOut } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+export function AggregatedTransactionsCallout() {
+  return (
+    <EuiCallOut
+      title={i18n.translate(
+        'xpack.apm.useAggregatedTransactions.callout.title',
+        {
+          defaultMessage:
+            'Transactions events are used to display data in this page',
```

Comment:
Would be good if the copy made it clear why the UI is now using transactions to render.

Perhaps:

```suggestion
            'No metrics events are available for the current time range so transaction events are used to display data instead',
```

Probably @bmorelli25 can help us here

Reply:
Is the average user going to understand _why_ this matters? If not, what's the takeaway? Does this mean data on the page is less accurate? Does the page load slower? Should the user adjust their search query?

Reply:
This warning is showed when we build the views with much more granular data, so it's possible they might notice a slower page load than expected. There shouldn't be much difference in the quality of data on display, but it's technically more accurate when this message is displayed, since it uses raw events.

Reply:
I think the problem now is that the title and paragraph are saying the same thing, just with different words.

I'm struggling to recommend a change because I don't understand what the user is supposed to do with this knowledge. If there's not much difference in the quality of data, is it even worth mentioning?

Reply:
> Is the average user going to understand why this matters?

Excellent question! For 7.15 it's not going to matter much. But starting in 8.0 we are planning to drop unsampled transactions.
This means that if the user have a sampling rate of 1% we are dropping 99% of transactions. Initially this will affect throughput (transactions per minute) the most. If the real throughput is 50tpm it'll only be displayed as 0.5tpm. We've talked about upscaling the number with a factor matching the sampling rate but let's just say that's another rabbit hole.

Dropping transactions shouldn't affect latency charts too much since the average of 1% should similar to the average of 100% - as long as we keep a statistical correct representation. Which is exactly what we are going to change with tail based sampling which will (primarily) sample slow transactions. 

The question is: why would anyone want to render anything with transactions if they are completely unreliable? Thanks for asking, yet another good question! If people want to see APM data for a particular user they'll use the kuery bar like `user.id: 1337`. We cannot support this use case with metrics because we've deliberately removed most dimensions to increase compression (and reduce storage costs). So if people really want to do this, they should be able to, although they should be aware of the drawbacks of transactions. The higher the sampling rate, the more inaccurate the results will be (generally speaking).

@ogupte @bmorelli25

Reply:
That's really interesting. Thanks–it makes a lot more sense now 😁. It sounds like we'll want to update this come 8.0, but for now, this is really all the user needs to know.

OK, my only qualm then becomes that the title and paragraph are too similar now:

```
ⓘ No metrics events were found in the current time range so transaction events are used to display data instead
This page is using data only from raw transaction events instead of aggregated transaction metrics.
```

Maybe get rid of the paragraph and just keep the title?

```
ⓘ This page is using transaction event data as no metrics events were found in the current time range
```

Or if that doesn't work, pick one for the title and explain the rest in the paragraph:

```
ⓘ No metrics events found
This page is using transaction event data as no metrics events were found in the current time range.
```